testing/smoke-test/src/smokeTest/groovy/org/gradle/play/integtest/external/PlayExternalContinuousBuildIntegrationTest.groovy

```diff
@@ -18,7 +18,9 @@ package org.gradle.play.integtest.external
 
 import org.gradle.play.integtest.fixtures.external.AbstractMultiVersionPlayExternalContinuousBuildIntegrationTest
 import org.gradle.play.integtest.fixtures.external.RunningPlayApp
+import spock.lang.Ignore
 
+@Ignore("https://github.com/gradle/gradle/issues/30530")
```

Comment:
🤔  Do we track this anywhere? We should probably add it to some Gradle 9 TODO list